R/drive_cp.R

```diff
@@ -0,0 +1,118 @@
+#' Copy a Google Drive file.
+#'
+#' `drive_copy()` is an alias for `drive_cp()`.
+#'
+#' @seealso Wraps the
+#' [drive.files.copy](https://developers.google.com/drive/v3/reference/files/copy)
```

Comment:
We should link to the API docs liberally. We will thank ourselves, when we're maintaining this.